### ISSUE TITLE
testit.sh fails on Cygwin and Ubuntu 20.04

### ISSUE THREAD

**Issue by lemniscati** [rpm-software-management/popt#58]
A simple build-and-check fails on Cygwin and Ubuntu 20.04:
```bash
git clone --depth 1 https://github.com/rpm-software-management/popt/ \
&& cd popt \
&& ./autogen.sh \
&& ./configure \
&& make \
&& make check \
&& echo OK || echo FAILURE
```

But, continued from the above, a modification by #57 make the tests pass.
```bash
git am <<'END' && make check && echo OK || echo FAILURE
From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Wed, 30 Dec 2020 21:39:45 +0900
Subject: [PATCH] tests/testit.sh: Fix expected results


diff --git a/tests/testit.sh b/tests/testit.sh
index 29228d9..b7edffa 100755
--- a/tests/testit.sh
+++ b/tests/testit.sh
@@ -115,7 +115,7 @@ run test1 "test1 - 56" "arg1: 0 arg2: (none) aFlag: 0xface" --nobitclr
 run test1 "test1 - 57" "arg1: 0 arg2: (none) aBits: foo,baz" --bits foo,bar,baz,!bar
 
 run test1 "test1 - 58" "\
-Usage: lt-test1 [-I?] [-c|--cb2=STRING] [--arg1] [-2|--arg2=ARG]
+Usage: test1 [-I?] [-c|--cb2=STRING] [--arg1] [-2|--arg2=ARG]
         [-3|--arg3=ANARG] [-onedash] [--optional=STRING] [--val]
         [-i|--int=INT] [-s|--short=SHORT] [-l|--long=LONG]
         [-L|--longlong=LONGLONG] [-f|--float=FLOAT] [-d|--double=DOUBLE]
@@ -124,7 +124,7 @@ Usage: lt-test1 [-I?] [-c|--cb2=STRING] [--arg1] [-2|--arg2=ARG]
         [-x|--bitxor] [--nstr=STRING] [--lstr=STRING] [-I|--inc]
         [-c|--cb=STRING] [--longopt] [-?|--help] [--usage] [--simple=ARG]" --usage
 run test1 "test1 - 59" "\
-Usage: lt-test1 [OPTION...]
+Usage: test1 [OPTION...]
       --arg1                      First argument with a really long
                                   description. After all, we have to test
                                   argument help wrapping somehow, right?
END
```

And, I don't know where `lt-` should come from...

**Comment by cgzones** [comment]
The `lt-` prefix is prepended by the specific libtool version in the Fedora CI.

**Comment by lemniscati** [comment]
@cgzones I've understood. Thank you.

Is there any way how to detect whether libtool is the specific version of not?

**Comment by pmatilai** [comment]
This was actually addressed in commit 043177829c3a32c9adea22355aa5acfb383af383 but didn't remember there was an open ticket on this too.
